<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QProgressBar" name="Battery">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QStackedWidget" name="Pages">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>401</width>
      <height>251</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="HomeScreen">
     <widget class="QPushButton" name="BolusOption">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>191</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Bolus</string>
      </property>
     </widget>
     <widget class="QPushButton" name="OptionsButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>191</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Options</string>
      </property>
     </widget>
     <widget class="QLabel" name="InsulinOnBoardLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Insulin On Board (IOB)</string>
      </property>
     </widget>
     <widget class="QLabel" name="UnitsLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Units:</string>
      </property>
     </widget>
     <widget class="QLabel" name="UnitsValue">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>170</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Text</string>
      </property>
     </widget>
     <widget class="QLabel" name="TimeRemainingLabel">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>170</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TimeRemaining:</string>
      </property>
     </widget>
     <widget class="QLabel" name="TimeRemainingValue">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>170</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Text</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="BolusScreen">
     <widget class="QLCDNumber" name="UnitsLCD">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>50</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Units</string>
      </property>
     </widget>
     <widget class="QLabel" name="CarbsBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>CARBS</string>
      </property>
     </widget>
     <widget class="QLabel" name="GlucoseLabel">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>70</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>GLUCOSE</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="CarbsSpinBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="GlucoseSpinBox">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>100</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="ViewCalculationButton">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>180</y>
        <width>161</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>View Calculation</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BolusBackButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;-</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ConfirmButton">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>10</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="OptionScreen">
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>171</width>
        <height>211</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>History</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="StopInsulinButton">
         <property name="text">
          <string>STOP INSULIN</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="LoadButton">
         <property name="text">
          <string>Load</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="TempRateButton">
         <property name="text">
          <string>Temp Rate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PersonalProfilesButton">
         <property name="text">
          <string>Personal Profiles</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Pump Settings</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="PersonalProfileScreen">
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>160</width>
        <height>171</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>Weekday</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QLCDNumber" name="Time">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="BolusLevel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>70</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="Date">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Date</string>
    </property>
   </widget>
   <widget class="QLabel" name="AmPmLabel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
